webapp/src/utils.ts

```diff
@@ -77,6 +77,15 @@ export function getChannelURL(state: GlobalState, channel: Channel, teamId: stri
     return channelURL;
 }
 
+export function shouldRenderCallsIncoming() {
+    const win = window.opener ? window.opener : window;
+    if (win.desktop && window.location.pathname.indexOf('/messages/') === -1) {
```

Comment:
Same, `includes()` would be a little easier on the brain.

Reply:
At least I'm consistent...